contracts/core/native_fallback/L2/NativeProver.sol

```diff
@@ -33,18 +33,6 @@ import {
 import {ProverHelpers} from "../../../libs/ProverHelpers.sol";
 
 contract NativeProver is Ownable, INativeProver {
-    /**
-     * @notice Stores proven block data for a chain
-     * @param blockNumber Number of the proven block
-     * @param blockHash Hash of the proven block
-     * @param stateRoot State root of the proven block
-     */
-    struct BlockProof {
```

Comment:
Removal of this makes sense as it was the struct we were previously using to cache proven blocks in the async approach.